labs/lab07.md

```diff
@@ -15,7 +15,7 @@ Clone the TicTacToe repository from: https://github.com/CMU-17-214/f22-rec07
 
```

Comment:
Can you look at the TicTacToe repository and see if any dependencies or source files need to be updated? We're using a new module configuration now and it would be nice to be consistent (e.g., `import` isn't a thing in ESM unlike CommonJS, it uses `require` instead). For reference, check the tsconfig file from homework 1. I want people to be able to directly apply what they do here to Homework 6b without having to figure out the difference in module systems.